uk.co.compucorp.civicrm.tasksassignments/views/dashboard/documents.html

```diff
@@ -70,8 +70,7 @@ <h3 class="panel-title">{{list.label[list.filterParams.due]}}</h3>
                 All
               <span>
                 ({{
-                  (list.list |
-                  filterByDateField:list.filterParams.due:list.filterParams.dateRange |
+                  ((list.filterByDateField(list.filterParams.due)) |
```

Comment:
You do not need `(list.filterByDateField(list.filterParams.due))`, you can do `list.filterByDateField(list.filterParams.due)`